docs/resources/aviatrix_spoke_gateway.md

```diff
@@ -255,13 +269,16 @@ The following arguments are supported:
 * `ha_gw_size` - (Optional) HA Gateway Size. Mandatory if enabling HA.
 * `ha_availability_domain` - (Optional) HA gateway availability domain. Required and valid only for OCI. Available as of provider version R2.19.3.
 * `ha_fault_domain` - (Optional) HA gateway fault domain. Required and valid only for OCI. Available as of provider version R2.19.3.
+* `manage_ha_gateway` - (Optional) Enable to manage Aviatrix spoke ha gateway using the aviatrix_spoke_gateway resource. If this is set to false, spoke ha gateways must be managed using the aviatrix_spoke_ha_gateway resource. Valid values: true, false. Default value: true. Available in provider R3.0+.
```

Comment:
when mentioning HA gateways, capitalize "HA"  since they are acronyms, in order to avoid confusion